src/views/Farms/Farms.tsx

```diff
@@ -192,16 +174,73 @@ const Farms: React.FC = () => {
     setQuery(event.target.value)
   }
 
-  let farmsStaked = []
-  if (isActive) {
-    farmsStaked = stakedOnly ? farmsList(stakedOnlyFarms) : farmsList(activeFarms)
-  } else {
-    farmsStaked = stakedOnly ? farmsList(stakedInactiveFarms) : farmsList(inactiveFarms)
-  }
+  const loadMoreRef = useRef<HTMLDivElement>(null)
+
+  const [numberOfItemsToShow, setNumberOfItemsToShow] = useState(12)
+  const [observerIsSet, setObserverIsSet] = useState<boolean>()
```

Comment:
Initialise to `false` please, and the type will be inferred from the default value, so no need for `boolean`